src/model/reference.ml

```diff
@@ -327,7 +414,12 @@ let parse whole_reference_location s :
     in
 
     match tokens with
-    | [] -> `Root (identifier, kind)
+    | [] -> (
+        match kind with
+        | #Paths.Reference.tag_any as kind -> `Root (identifier, kind)
+        | `TPathComponent -> `Page_path (`TRelativePath, [ identifier ]))
```

Comment:
Do I understand correctly that this can only happen in case of `/` ending references, such as `{!a/}` or `{!/}`? If so, I think that those references are currently not allowed.

If we allow them, shouldn't it be `` `Any_path ``? and what about `identifier = ""` ? :)

Reply:
Indeed that's not allowed by the lexer. I'll put `assert false` here.